### PR TITLE
AWS Release candidate

### PR DESCRIPTION
**Pull request by cuongquangnam**

### CODE REVIEW

**Comment by chloezxyy** [comment]
I saw that you're manually uploading the zipfile, but im not sure if we want to do that manually. Have to check with either @thedoublejay or @veralygit

**Review by pierregee — CHANGES_REQUESTED** [review]

**Comment by veralygit** [comment]
there is 2 ways to go about this. Are you going to package up this code and the lambda function calls your code from the package on the ops side 

or are we looking to shifting the codes here to the the lambda function directly

**Comment by cuongquangnam** [comment]
by shifting the codes to lambda function directly, you mean copy and paste the index.ts file to the lambda ? 

> there is 2 ways to go about this. Are you going to package up this code and the lambda function calls your code from the package on the ops side
> 
> or are we looking to shifting the codes here to the the lambda function directly

by shifting the codes to lambda function directly, you mean copy and paste index.ts to lambda?

File: README.md
Comment (veralygit) [inline README.md:1]: haha need some enlightenment on why we need to zip the file

**Pull request closed without merging**